src/main/kotlin/com/lovelysystems/gradle/S3UploadDirectory.kt

```diff
@@ -24,6 +24,7 @@ import java.util.concurrent.ExecutionException
 abstract class S3UploadDirectory : DefaultTask() {
 
     @get:Input
+    @Deprecated("This property has no effect, the profile configured in awsProject(<profile>, <region>) is used.")
```

Comment:
we could keep profile and region as optional to allow override the globals, wdyt?

Reply:
added .*Override to both s3 tasks, I think its a better name. Kept the original properties here but deprecated